### PR TITLE
Update main.py for new API Catalog models

### PR DESCRIPTION
**Pull request by rohrao**
Added changes for the new API Catalog models

### CODE REVIEW

**Comment by rohrao** [comment]
@dglogo @shubhadeepd to review

File: community/5_mins_rag_no_gpu/main.py [hunk 1]
```diff
@@ -22,56 +22,44 @@
...
             st.success(f"File {uploaded_file.name} uploaded successfully!")
-            with open(os.path.join(DOCS_DIR, uploaded_file.name),"wb") as f:
+            with open(os.path.join(DOCS_DIR, uploaded_file.name), "wb") as f:
                 f.write(uploaded_file.read())
 
-############################################
 # Component #2 - Embedding Model and LLM
-############################################
+llm = ChatNVIDIA(model="meta/llama3-70b-instruct")
+document_embedder = NVIDIAEmbeddings(model="NV-Embed-QA", model_type="passage")
```
Comment (dglogo) [inline community/5_mins_rag_no_gpu/main.py:53]: NV-Embed-QA -> nvidia/nv-embedqa-e5-v5 (that's the new default)

Comment (rohrao) [reply community/5_mins_rag_no_gpu/main.py]: Ack, fixed this and the requirements.txt

**Review by dglogo — APPROVED** [review]
thanks for bringing it back!

**Pull request merged**